bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/RequestClient.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Altinn.Authentication.UI.Core.Authentication;
+using Altinn.Authentication.UI.Core.SystemUsers;
+using Altinn.Authentication.UI.Integration.AccessToken;
+using Altinn.Authentication.UI.Integration.Configuration;
+using Altinn.Authorization.ProblemDetails;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using Altinn.Authentication.UI.Core.Extensions;
+using System.Text.Json;
```

Comment:
i dont see any serializing casing options but i see you have defined camelcasing on the properties

Reply:
ok fixed

Reply:
ok, fixed